bucket/lemminx.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "0.21.0",
+    "description": "LeMMinX is a language server for XML and XSD files",
+    "homepage": "https://github.com/redhat-developer/vscode-xml",
```

```suggestion
    "homepage": "https://github.com/eclipse/lemminx",
```

We can keep this as homepage as this is the original source